Project-Bootcamp/views/character-view.js

```diff
@@ -0,0 +1,78 @@
+var React = require ('react');
+var _ = require ('lodash');
+var $ = require ('jquery');
+var Button = require ('../components/button');
+var Input = require ('../components/input');
+var List = require ('../components/list');
+var BlizzardAPI = require ('../querys/query');
+
+var CharacterView = React.createClass ({
+
+    getInitialState: function () {
+
+        return {
+            character: {}
+        }
+    },
+
+    render: function () {
+
+        return (
+            <div>
+                <div>
+                    <h3><u>Character basic information form</u></h3>
+                    <p>Fill the form with the character name and realm.</p>
+                    <Input label='Name' type="text" onChange={this.handleInputChange.bind (this, 'name')} />
```

Comment:
**Minor:** use double quotes for properties. Also, you might think on breaking this into a new ``getWhateverProps()``, since you have 3 props in the same line.